app/controllers/schools/advice/advice_base_controller.rb

```diff
@@ -71,22 +72,22 @@ def set_page_title
         @advice_page_title = t("advice_pages.#{@advice_page.key}.page_title")
       end
 
-      def set_page_subtitle
-        @advice_page_subtitle = if_exists("#{action_name}.title")
+      def set_page_subtitle(section: nil)
+        @advice_page_subtitle = if_exists("#{action_name}.title", section:)
       end
 
       def set_breadcrumbs
         @breadcrumbs = [
           { name: t('advice_pages.breadcrumbs.root'), href: school_advice_path(@school) },
-          { name: @advice_page_title, href: advice_page_path(@school, @advice_page) },
+          { name: @advice_page_title, href: advice_page_path(@school, @advice_page) }
         ]
       end
 
-      def if_exists(key)
-        full_key = "advice_pages.#{@advice_page.key}.#{key}"
-        if I18n.exists?(full_key, I18n.locale)
-          t(full_key)
-        end
+      def if_exists(key, section: nil)
```

Comment:
What I've been doing in recent code is using `default: nil`, e.g. `I18n.t(full_key, default: nil)` avoids throwing an exception if key has not been set.